### ISSUE TITLE
Wrong location of interaction activity in NetworkRoutingInclAccessEgressModule?

### ISSUE THREAD

**Issue by sebhoerl** [matsim-org/matsim-libs#2520]
While debugging a relatively unrelated topic, I noticed some weird interaction activities in my simulation when using car trips with walk access/egress. So I had a look at the code and I think that there might be a bug (but maybe I'm missing something).

Usually we would want to have
```
[ origin activity location ] > walk > [ interaction location ] > car
```

Looking at the code, this is not really what happens. In `NetworkRoutingInclAccessEgressModule::calcRoute` we first obtain the `fromFacility` and `toFacility` from the trip origin and destination activities. After, the code assigns `accessActLink` and `egressActLink` based on these facilities. But this is not the intended behaviour. The access link should rather be the link that will be identified by the delegated `car` router as the initial link of the `car` route, shouldn't it? The same applies for the egress part.

Basically, this means that currently all access/egress walk legs to car are zero duration?

**Comment by sebhoerl** [comment]
Actually, I think it is fine. Something else is going on.
